addons/project_todo/models/project_task.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, _, _lt
+from odoo.tools import html2plaintext
+
+class Task(models.Model):
+    _inherit = 'project.task'
```

Comment:
@xavierbol I think you made a mistake, "project.task" is already used in another context, why not using a specific name like "note.note" (with less fields, better to save db usage) :disguised_face: ?

PS: don't hit me, please, I couldn't resist